### PR TITLE
halilozercan patch

### PR DESCRIPTION
**Pull request by halilozercan**
Please squash while merging.

### CODE REVIEW

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md [hunk 1]
```diff
@@ -2,15 +2,28 @@
...
 ---
 
 ### Education
+  
+- M.Sc., Computer Engineering, Spring, 2019
+  - [Bilkent University](http://www.cs.bilkent.edu.tr/), Ankara, Turkey 
+  - Thesis: ["A cryptocurrency incentivized voluntary grid computing platform for DNA read alignment"](https://tez.yok.gov.tr/UlusalTezMerkezi/tezDetay.jsp?id=GBP9_s_CeuzDAn-CaOlOmQ&no=-nn8esht_gQgWS7bJ_mYEw).
+  
+- B. Sc., Computer Engineering, 2016 
+  - [Bilkent University](), Ankara, Turkey
```
Comment (halilozercan) [inline team/_posts/2016-09-01-halil-ibrahim-özercan.md]: aga link ver

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md [hunk 2]
```diff
@@ -2,15 +2,28 @@
...
+
+### Publication
```
Comment (calkan) [inline team/_posts/2016-09-01-halil-ibrahim-özercan.md]: Publications

Comment (halilozercan) [reply team/_posts/2016-09-01-halil-ibrahim-özercan.md]: not if it is singular

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md (hunk 1 above)
Comment (calkan) [reply team/_posts/2016-09-01-halil-ibrahim-özercan.md]: evet link ver

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md [hunk 3]
```diff
@@ -2,15 +2,28 @@
...
+
+- [**Realizing the potential of blockchain technologies in genomics**](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120626/) **Halil Ibrahim Ozercan**, Atalay Mert Ileri, Erman Ayday, and Can Alkan. *Genome Res.* 2018 Sep; 28(9): 1255–1263. 
 
 ### Post Alkan Lab
- is now employed as a .
+ is now employed as a Software Engineer at Google in London, UK.
```
Comment (calkan) [inline team/_posts/2016-09-01-halil-ibrahim-özercan.md]: Halil is .... diye yazsana

**Review by calkan — CHANGES_REQUESTED** [review]
some comments

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md (hunk 3 above)
Comment (halilozercan) [reply team/_posts/2016-09-01-halil-ibrahim-özercan.md]: Halil is _implied_ 😎

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md (hunk 1 above)
Comment (halilozercan) [reply team/_posts/2016-09-01-halil-ibrahim-özercan.md]: verdim

File: team/_posts/2016-09-01-halil-ibrahim-özercan.md (hunk 2 above)
Comment (halilozercan) [reply team/_posts/2016-09-01-halil-ibrahim-özercan.md]: publications now

**Comment by halilozercan** [comment]
Addressed some comments

**Pull request merged**